docs/modules/servers/partials/RemoteDelivery.adoc

```diff
@@ -74,6 +74,9 @@ within the mailet code.<br>
 Note: This facility should be used with extreme care by expert users with a thorough knowledge of the relevant RFCs and
 the ability to perform their own problem resolutions.
 * *debug* (optional) - a Boolean (true/false) indicating whether debugging is on. Default is false.
+* *usePriority* (optional) - a Boolean value (true/false) that indicates whether email priority support is enabled.
+When this option is enabled, any email that does not have a specified priority will be assigned the highest priority upon
+delivery and the lowest priority during retry attempts. Default is false.
```

Comment:
If enabled, I think we should not communicate priorities for mails that do not have one.

Reply:
If this property `usePriority` is enabled but does not affect emails without priorities set, I don't quite understand what it is for. Based on my understanding, this property was initially designed to hardcode email priorities through configuration, with the highest priority assigned on send and the lowest on retry. To maintain the original intent while also supporting the new priority functionality, I propose using this property to assign priorities to emails that currently lack a specified priority. 

I also have a couple of alternative suggestions: 

1. Always assign the highest priority when sending and the lowest when retrying, irrespective of any priority specified in the email, if `usePriority` is enabled. . 
2. Eliminate this property entirely and only consider the priority if it is explicitly set in the email.

What are your thoughts on these options?

Reply:
> If this property `usePriority` is enabled but does not affect emails without priorities set, I don't quite understand what it is for. Based on my understanding, this property was initially designed to hardcode email priorities through configuration, with the highest priority assigned on send and the lowest on retry. To maintain the original intent while also supporting the new priority functionality, I propose using this property to assign priorities to emails that currently lack a specified priority.
> 
> I also have a couple of alternative suggestions:
> 
> 1. Always assign the highest priority when sending and the lowest when retrying, irrespective of any priority specified in the email, if `usePriority` is enabled. .
> 2. Eliminate this property entirely and only consider the priority if it is explicitly set in the email.
> 
> What are your thoughts on these options?

I chose the third option. I left only the lowest priority for retry attempts.

Reply:
> only consider the priority if it is explicitly set in the email.

This is what I meant.

The property is however useful when set to false as it allows tiurning off priorities when dealing with unknown third party systems when performing remote delivery mx resolution.

(Prioritiesmakes sens localy or with known 3rd party)